api/v4/openapi.yaml

```diff
@@ -413,4 +481,95 @@ components:
         ref:
           $ref: '#/components/schemas/PackageRef'
         reason:
-          type: string
\ No newline at end of file
+          type: string
+    ModelCardRequest:
+      type: object
+      properties:
+        models:
+          type: array
+          items:
+            type: object
+            properties:
+              name:
+                type: string
+                description: Name of the model to search for
+              filter:
+                type: object
+                description: Arbitrary key/value pairs to filter models with the same name
+                additionalProperties:
+                  type: string
+    ListModelCardResponse:
+      type: object
+      properties:
+        id:
+          type: string
+          description: Unique identifier for the model card
+        name:
+          type: string
+          description: Name of the model
+        tasks:
+          type: array
+          items:
+            $ref: '#/components/schemas/TaskSummary'
+        tags:
+          type: array
+          items:
+            type: string
+            description: Arbitrary tags associated with the model
+        config:
+          type: object
+          description: Context configuration for the report
+          properties:
+            ref:
+              $ref: '#/components/schemas/ReportConfig'
+    TaskSummary:
+      type: object
+      properties:
+        task:
+          type: string
+          description: Task name evaluated
+        metric:
+          type: string
+          description: Metric name evaluated
+        friendlyName:
+          type: string
+          description: Friendly name of the metric
```

Comment:
What is the difference between `task` and `metric`? Im assuming task == 'crows_pairs_english' etc and  friendlyName == 'CrowS-Pairs (stereotyping)', not sure what metric means though

Reply:
the metric is what is measured in each task:

```
    "bbq": {
      "alias": "bbq",
      "acc,none": 0.4653627846543117,
      "acc_stderr,none": 0.0020624379655692247,
      "accuracy_amb,none": 0.09413253094440266,
    }
```
The task is `bbq` and the metrics are `acc,none`, `acc_stderr,none`, `accuracy_amb,none`, etc.